build/gulpfile.js

```diff
@@ -0,0 +1,34 @@
+const gulp = require('gulp');
```

Comment:
since you are building a library it may be a good opportunity to try rollup. What do you think?

Reply:
I am using it in other projects and like `gulp` more.